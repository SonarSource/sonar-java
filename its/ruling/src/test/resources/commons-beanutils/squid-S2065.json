{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/BaseDynaBeanMapDecorator.java':[
56,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/BeanMap.java':[
50,
52,
53,
54,
],
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils/converters/AbstractConverter.java':[
72,
],
}
