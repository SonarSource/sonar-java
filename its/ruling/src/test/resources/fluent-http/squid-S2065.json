{
'net.code-story:http:src/main/java/net/codestory/http/misc/MemoizingSupplier.java':[
24,
27,
],
}
