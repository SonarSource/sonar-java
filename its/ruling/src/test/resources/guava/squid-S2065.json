{
'com.google.guava:guava:src/com/google/common/base/Converter.java':[
99,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMapBasedMultimap.java':[
1263,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMultimap.java':[
104,
148,
160,
172,
211,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractMultiset.java':[
144,
175,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractSortedMultiset.java':[
120,
],
'com.google.guava:guava:src/com/google/common/collect/AbstractTable.java':[
107,
163,
],
'com.google.guava:guava:src/com/google/common/collect/CartesianList.java':[
37,
38,
],
'com.google.guava:guava:src/com/google/common/collect/DescendingMultiset.java':[
37,
48,
109,
],
'com.google.guava:guava:src/com/google/common/collect/MapConstraints.java':[
301,
],
'com.google.guava:guava:src/com/google/common/collect/Maps.java':[
3377,
3384,
3395,
3889,
3985,
4016,
],
'com.google.guava:guava:src/com/google/common/collect/Sets.java':[
1157,
1158,
],
'com.google.guava:guava:src/com/google/common/collect/TreeBasedTable.java':[
248,
],
'com.google.guava:guava:src/com/google/common/collect/TreeRangeSet.java':[
70,
71,
248,
],
'com.google.guava:guava:src/com/google/common/io/BaseEncoding.java':[
749,
750,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/AtomicLongMap.java':[
282,
],
}
