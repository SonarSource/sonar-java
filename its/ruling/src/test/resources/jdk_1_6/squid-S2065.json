{
'jdk_1_6:java/awt/KeyboardFocusManager.java':[
94,
355,
],
'jdk_1_6:java/awt/Robot.java':[
140,
154,
],
'jdk_1_6:java/awt/SystemTray.java':[
106,
],
'jdk_1_6:java/awt/TrayIcon.java':[
83,
85,
86,
87,
92,
],
'jdk_1_6:java/awt/font/FontRenderContext.java':[
50,
51,
52,
53,
],
'jdk_1_6:java/awt/geom/Path2D.java':[
80,
81,
82,
83,
],
'jdk_1_6:java/beans/Beans.java':[
610,
611,
612,
613,
614,
],
'jdk_1_6:java/lang/ref/Reference.java':[
78,
],
'jdk_1_6:java/lang/reflect/Constructor.java':[
53,
55,
610,
],
'jdk_1_6:java/lang/reflect/Field.java':[
51,
53,
1010,
],
'jdk_1_6:java/lang/reflect/Method.java':[
56,
58,
689,
],
'jdk_1_6:java/security/cert/X509CRL.java':[
97,
],
'jdk_1_6:java/text/MergeCollation.java':[
192,
193,
197,
207,
],
'jdk_1_6:java/util/AbstractList.java':[
589,
],
'jdk_1_6:java/util/AbstractMap.java':[
282,
283,
],
'jdk_1_6:java/util/TreeMap.java':[
1496,
1496,
],
'jdk_1_6:java/util/WeakHashMap.java':[
807,
],
'jdk_1_6:java/util/concurrent/DelayQueue.java':[
42,
43,
],
'jdk_1_6:java/util/concurrent/SynchronousQueue.java':[
538,
540,
546,
],
}
