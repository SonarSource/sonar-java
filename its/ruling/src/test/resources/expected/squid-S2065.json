{
'project:com/sun/corba/se/impl/ior/ObjectReferenceProducerBase.java':[
43,
],
'project:com/sun/corba/se/impl/naming/cosnaming/NamingContextImpl.java':[
999,
],
'project:com/sun/corba/se/impl/presentation/rmi/StubInvocationHandlerImpl.java':[
73,
74,
75,
76,
],
'project:com/sun/corba/se/impl/util/IdentityHashtable.java':[
59,
64,
],
'project:com/sun/java/swing/plaf/windows/WindowsToggleButtonUI.java':[
115,
116,
117,
],
'project:com/sun/jmx/interceptor/DefaultMBeanServerInterceptor.java':[
117,
121,
125,
128,
134,
],
'project:com/sun/jmx/snmp/daemon/CommunicatorServer.java':[
179,
192,
203,
225,
228,
230,
231,
235,
238,
239,
241,
],
'project:com/sun/jmx/snmp/daemon/SnmpAdaptorServer.java':[
197,
198,
199,
200,
201,
202,
203,
208,
],
'project:com/sun/jmx/snmp/daemon/SnmpInformRequest.java':[
109,
114,
],
'project:com/sun/jmx/snmp/daemon/SnmpRequestHandler.java':[
74,
75,
76,
81,
86,
88,
89,
90,
91,
],
'project:com/sun/jmx/snmp/daemon/SnmpSession.java':[
52,
57,
61,
67,
75,
80,
],
'project:com/sun/jmx/snmp/daemon/SnmpSocket.java':[
43,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/Hashtable.java':[
60,
61,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTM.java':[
79,
89,
98,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/sax2dtm/SAX2DTM.java':[
94,
99,
104,
110,
113,
119,
122,
125,
128,
],
'project:com/sun/org/apache/xpath/internal/NodeSet.java':[
720,
772,
776,
812,
],
'project:com/sun/org/apache/xpath/internal/compiler/XPathParser.java':[
56,
63,
],
'project:java/awt/KeyboardFocusManager.java':[
152,
412,
],
'project:java/awt/Robot.java':[
181,
195,
],
'project:java/awt/SystemTray.java':[
126,
],
'project:java/awt/TrayIcon.java':[
103,
105,
106,
107,
],
'project:java/awt/font/FontRenderContext.java':[
68,
69,
70,
71,
],
'project:java/awt/geom/Path2D.java':[
97,
98,
99,
100,
],
'project:java/beans/Beans.java':[
590,
591,
592,
593,
594,
],
'project:java/lang/ref/Reference.java':[
95,
],
'project:java/lang/reflect/Constructor.java':[
71,
73,
625,
],
'project:java/lang/reflect/Field.java':[
69,
71,
1029,
],
'project:java/lang/reflect/Method.java':[
74,
76,
695,
],
'project:java/security/cert/X509CRL.java':[
119,
],
'project:java/text/MergeCollation.java':[
209,
210,
214,
224,
],
'project:java/util/AbstractList.java':[
601,
],
'project:java/util/AbstractMap.java':[
299,
300,
],
'project:java/util/TreeMap.java':[
1517,
1517,
],
'project:java/util/WeakHashMap.java':[
830,
],
'project:java/util/concurrent/DelayQueue.java':[
71,
],
'project:java/util/concurrent/SynchronousQueue.java':[
598,
600,
606,
],
'project:javax/management/modelmbean/RequiredModelMBean.java':[
139,
],
'project:javax/management/timer/Timer.java':[
145,
151,
],
'project:javax/swing/MenuSelectionManager.java':[
74,
],
'project:javax/swing/SortingFocusTraversalPolicy.java':[
81,
82,
],
'project:javax/swing/ToolTipManager.java':[
61,
],
'project:javax/swing/plaf/basic/BasicPopupMenuUI.java':[
70,
],
'project:javax/swing/plaf/basic/BasicScrollBarUI.java':[
1062,
1063,
1063,
1064,
],
'project:javax/swing/plaf/basic/BasicSliderUI.java':[
70,
1567,
1568,
1568,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
164,
],
'project:javax/swing/plaf/basic/BasicTextUI.java':[
1302,
1303,
1304,
1305,
],
'project:javax/swing/plaf/basic/BasicTreeUI.java':[
66,
67,
95,
102,
],
'project:javax/swing/plaf/metal/MetalBumps.java':[
148,
],
'project:javax/swing/plaf/synth/SynthSliderUI.java':[
80,
81,
],
'project:javax/swing/tree/DefaultTreeCellEditor.java':[
78,
91,
94,
97,
100,
106,
112,
],
'project:org/omg/CORBA/portable/ObjectImpl.java':[
57,
],
'project:org/omg/PortableServer/Servant.java':[
44,
],
}
